docker/scripts/compile-docker.sh

```diff
@@ -51,7 +51,7 @@ echo "Running build in container"
 docker run \
     --rm \
     -e TARGET_PLATFORM=$TARGET_PLATFORM \
-    -e SCRATCH_DIR="/scratch" \
+    -e SCRATCH_DIR="/$HERON_VERSION" \
```

Comment:
This is not the right fix. SCRATCH_DIR is used to create directory

Reply:
In status.sh, version string is decided in two steps:
1. if [ -d .git ];, find the branch name and use it as version string,
2. otherwise, use the current directory name as version string.

In docker script, .git file is not available because tarball doesn't include this hidden file. This directory name is used as version number.